qa-pipelines/tasks/usb-deploy.sh

```diff
@@ -63,23 +63,50 @@ DB_EXTERNAL_IP=$(kubectl get nodes -o json | jq -r '[.items[] | .status.addresse
 
 helm init --client-only
 
-helm install stable/postgresql \
-  --namespace postgres         \
-  --name postgres              \
-  --set "service.externalIPs={${DB_EXTERNAL_IP}}"
+is_namespace_ready() {
+  for (( i = 0  ; i < 120 ; i ++ )) ; do
```

Comment:
extra for loop is unnecessary because we're already looping in the calling function (with sleep 10 also)